assets/js/blocks/cart-checkout/mini-cart-contents/edit.tsx

```diff
@@ -43,7 +43,11 @@ interface Props {
 }
 
 const Edit = ( { clientId }: Props ): ReactElement => {
-	const blockProps = useBlockProps();
+	const blockProps = useBlockProps( {
+		style: {
+			minHeight: '100vh',
+		},
+	} );
```

Comment:
Can we move it to css at https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/d67e5191f0265a9ea8eb02aa274b9d3f459335ff/assets/js/blocks/cart-checkout/mini-cart-contents/editor.scss#L1-L6

We already have the `max-width` styles and other editor specific styles living there.

Reply:
On `trunk, if you switch back and forth editing the template part, the editor height is correct the second time.

Reply:
> Would it be worth to document it with a comment so whoever stumbles upon it doesn't move it to styles in the future?

@tomasztunik I added the comment explaining the usage of the inline style.

Reply:
thanks!